README.md

```diff
@@ -200,6 +200,43 @@ const player = new Player({
 });
 ```
 
+### Setting up a playback configurations
+
+If needed, the default player behavior can be configured through the `playbackConfig` key when initialized.
+
+
+```typescript
+// Simply pass the `playbackConfig` property to `PlaybackConfig` when instantiating a player.
```

Comment:
```
// Simply pass the `playbackConfig` property to `PlayerConfig` when instantiating a player.
```
This should be the `PlayerConfig` or? 🤔 
(cc @pogist)

Reply:
yes, that should be `PlayerConfig`